packages/lune/src/builder/rolldown/lightningcss.ts

```diff
@@ -0,0 +1,32 @@
+import type { Plugin } from "rolldown";
```

Comment:
LightningCSS does not support SCSS which makes it just *not enough* to be a viable dropin replacement, but I am open to using it as a CSS Modules implementation if the plugin providing scss does not already provide it, or moving our scss implementation to one based on lightning if such a thing exists.

Reply:
I'll write a scss plugin for rolldown which should compliment this.